src/main/java/wtf/g4s8/examples/configuration/TransactionTest.java

```diff
@@ -0,0 +1,123 @@
+package wtf.g4s8.examples.configuration;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import wtf.g4s8.examples.spaxos.*;
+import wtf.g4s8.examples.system.*;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+@Builder
+@AllArgsConstructor
+public class TransactionTest {
+    private final int nUpdaters;
+    private final int nReplicas;
+    private final ExecutorService accexec;
+    private final boolean withDrops;
+    private final double dropRate;
+    private final boolean withTimeout;
+    private final int timeout;
+    private final boolean async;
+    private final int syncDelay;
+
+
+    public void test() throws Exception {
+        writeConcurrent(
+                nUpdaters,
+                new StupidTransactionManager(
+                        syncDelay,
+                        resourceManagerCluster()
+                )
+        );
+    }
+
+
+    private void writeConcurrent(int nUpdaters, StupidTransactionManager tm) throws Exception {
+        CountDownLatch cd = new CountDownLatch(1);
+        CountDownLatch done = new CountDownLatch(nUpdaters);
+        ExecutorService exec = Executors.newCachedThreadPool();
+        List<String> transactionsID =
+                Stream.generate(UUID::randomUUID)
```

Comment:
@stepanvalyavskiy can we use some smaller ids to make it moe human-readable in logs? I think incrementing integer would be enough here